parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs

```diff
@@ -191,8 +185,8 @@ pub type Barrier = TrailingSetTopicAsId<
 				(
 					// If the message is one that immediately attemps to pay for execution, then allow it.
 					AllowTopLevelPaidExecutionFrom<Everything>,
-					// Parent, its pluralities (i.e. governance bodies) and system parachains get free execution.
-					AllowExplicitUnpaidExecutionFrom<(ParentOrParentsPlurality, SystemParachains)>,
+					// Parent and its pluralities (i.e. governance bodies) get free execution.
```

Comment:
no, no, governance calls are unpaid and covered by `AllowExplicitUnpaidExecutionFrom<(ParentOrParentsPlurality,)>,`

we have there several tests e.g. `change_storage_constant_by_governance_works` for several `pub storage` or `initialize_bridge_by_governance_works`, all of those calls uses [`execute_as_governance`](https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/test-utils/src/lib.rs#L305-L319) which does `UnpaidExecution/Transact`

I thought that it will be more clear if we start just with paid version for all parachains, also our router on AH is paid so does not make sense to allow unpaid for AH, I dont know, maybe as some kind of fallback for some reason?

I hope we get [`FeeManager`](https://github.com/paritytech/polkadot/pull/7005) in soon, otherwise we will be burning fees